aiohttp/web_urldispatcher.py

```diff
@@ -296,7 +264,11 @@ def get_info(self):
         return {'path': self._path}
 
     def url(self, *, query=None):
-        return self._append_query(self._path, query)
+        super().url()
```

Comment:
Warning here will suggest to use `url_for`, but `url_for` doesn't accepts `query` parameter. So deprecation message should be a bit different or it will cause confusion.